src/mappings/core.ts

```diff
@@ -46,7 +46,7 @@ export function handleTransfer(event: Transfer): void {
 
   // get pair and load contract
   let pair = Pair.load(event.address.toHexString())
-  let pairContract = PairContract.bind(event.address)
+  // let pairContract = PairContract.bind(event.address)
```

Comment:
@paulbarclay It is used at line #196

Reply:
Right. I re-added the use, but not the contract. I rely on intellisense so much.